obspy/core/event/base.py

```diff
@@ -780,7 +780,7 @@ def get_quakeml_uri(self, authority_id="local"):
             id = str(uuid4())
 
         regex = r"^(smi|quakeml):[\w\d][\w\d\-\.\*\(\)_~']{2,}/[\w\d\-\." + \
-                r"\*\(\)_~'][\w\d\-\.\*\(\)\+\?_~'=,;#/&amp;]*$"
+                r"\*\(\)_~'][\w\d\-\.\*\(\)\+\?_~'=,;#/&]*$"
```

Comment:
Should be corrected also in the docstring (line 441)

Reply:
Done directly in master: 4039c69973106b340477ab23495be4e8270a3f84